BreachHigh.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+#include "BatteryChecker.h"
+
+extern tst_BatteryStatus batteryStatus_st;

```

Comment:
Avoid globals - it will prevent this from running in multiple threads. It also makes it hard to track changes to this variable. Use return values instead.